js/script.js

```diff
@@ -99,13 +169,65 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
   const title = document.querySelector('#title').value;
   const author = document.querySelector('#author').value;
   const isbn = document.querySelector('#isbn').value;
+  const email = document.querySelector('#authorEmail').value;
+  const phone = document.querySelector('#author-phone').value;
 
   // Validate
-  if(title === '' || author === '' || isbn === '') {
-    UI.showAlert('Please fill in all fields', 'danger');
+  if (title === '') {
+    UI.show_error_text('.titleError');
+  }
+  if (author === '') {
+    UI.show_error_text('.authorError');
+  }
+  if (isbn === '') {
+    UI.show_error_text('.isbnError');
```

Comment:
You can use regexes to validate input more accurately. For example: [here are some ideas](https://www.regexlib.com/Search.aspx?k=ISBN)